net-analyzer/openvas/openvas-9.0.0.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit readme.gentoo-r1
+
+DESCRIPTION="A remote security scanner"
+HOMEPAGE="http://www.openvas.org/"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+IUSE="+extras +cli +gsa +ospd ldap radius"
+
+DEPEND="
+	radius? ( >=net-analyzer/openvas-libraries-9.0.3[radius] )
+	ldap? ( >=net-analyzer/openvas-libraries-9.0.3[ldap] )
+	>=net-analyzer/openvas-libraries-9.0.3
+	>=net-analyzer/openvas-scanner-5.1.3
+	>=net-analyzer/openvas-manager-7.0.3
+	gsa? ( >=net-analyzer/greenbone-security-assistant-7.0.3 )
+	cli? ( >=net-analyzer/gvm-tools-1.4.1 )
+	ospd? ( >=net-analyzer/ospd-1.3.2 )
+	extras? (
+		>=net-analyzer/openvas-libraries-9.0.3[extras]
+		>=net-analyzer/openvas-scanner-5.1.3[extras]
+		>=net-analyzer/openvas-manager-7.0.3[extras]
+		>=net-analyzer/greenbone-security-assistant-7.0.3[extras]
+		>=net-analyzer/ospd-1.3.2[extras]
```

Comment:
Make a new condition for extras? ( ospd? ...

same for greenbone-security-assistant

Reply:
This is not clear for me.
Can you explain a bit?

Reply:
Currently, ospd is pulled when 'ospd' USE flag is enabled. However, it will pull it also with 'extras'. So you'd need to move this line under 
```
ospd? ( 
	>=net-analyzer/ospd-1.3.2 
	extras? ( >=net-analyzer/ospd-1.3.2[extras] )
)
```

Similarily with greenbone-security-assistant. This is fine-tuning the logic :P